src/routes/safe/components/Transactions/TxList/TxDetails.tsx

```diff
@@ -91,6 +93,21 @@ export const TxDetails = ({ transaction }: TxDetailsProps): ReactElement => {
   const willBeReplaced = txStatus === LocalTransactionStatus.WILL_BE_REPLACED
   const isPending = txStatus === LocalTransactionStatus.PENDING
   const currentUser = useSelector(userAccountSelector)
+  const hasModule = transaction.txDetails && isModuleExecutionInfo(transaction.txDetails.detailedExecutionInfo)
+  const isMultiSend = data && isMultiSendTxInfo(data.txInfo)
+
+  // To avoid prop drilling into TxDataGroup, module details are positioned here accordingly
+  const getModuleDetails = () => {
+    if (!transaction.txDetails || !isModuleExecutionInfo(transaction.txDetails.detailedExecutionInfo)) {
+      return null
+    }
+
+    return (
+      <div className="tx-module">
```

Comment:
Class doesn't seem to be targeted.

Reply:
A few others aren't here either. I added it in case we do want to in the future. Do you think it should not be?

Reply:
Let's leave it and tackle all at the same time.